src/screens/TopicPageScreen/MemoizedListComponent.js

```diff
@@ -224,7 +223,8 @@ const RenderListFeed = (props) => {
           withToast={true}
           isVisible={isBlurred}
           containerStyle={{
-            height: commentHeight()
+            height: isBlurred || !hasComment ? addCommentPreviewHeight() : undefined,
+            marginBottom: hasComment ? normalize(4) : 0
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The height of the container is now set based on whether the post is blurred or has no comments. This seems to be a design change, but please ensure that this change doesn't affect other parts of the layout.
```diff
- height: isBlurred || !hasComment ? addCommentPreviewHeight() : undefined,
- marginBottom: hasComment ? normalize(4) : 0
+ height: commentHeight()
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->